docs/test/anchors.test.js

```diff
@@ -0,0 +1,23 @@
+var fs = require('fs');
```

Comment:
Rename this file to `__test__/anchors.test.js` (see other comment about `package.json`)

Reply:
oh I get it